web/src/message_list_view.js

```diff
@@ -281,10 +281,14 @@ export class MessageListView {
         if (last_edit_timestamp !== undefined) {
             const last_edit_time = new Date(last_edit_timestamp * 1000);
             const today = new Date();
+            let date = timerender.render_date(last_edit_time, today)[0].textContent;
+            if (["Today", "Yesterday"].includes(date)) {
```

Comment:
Is the `["Today", "Yesterday"]` array going to work with internationalization and other language strings? A quick check against German in development suggests it's not: "Last edited Heute um 2:47PM Uhr" (and while German does capitalize nouns, _heute_ is an adverb, and therefore would not be capitalized).

Perhaps what you need to do is try and match the string against the presence of a digit? Something like `if (!date.match(/\d/) {`, which will be `null` if there's no digit, which might be safe to assume is an adverb like "today", "heute", "hoy" (Spanish), etc.

Reply:
Thanks a lot for catching this bug. I didn't realize that the date I was obtaining was already internationalized in a different language, so "today" and "tomorrow" wouldn't work. thanks a lot for your solution. I have updated the code with the regex, and now it is working as expected regardless of the language.